harvester_e2e_tests/apis/test_images.py

```diff
@@ -27,12 +27,17 @@
 ]
 
 
+@pytest.mark.images_p1
+@pytest.mark.p1
 def test_list_images(admin_session, harvester_api_endpoints):
     resp = admin_session.get(harvester_api_endpoints.list_images)
     assert resp.status_code == 200, (
         'Failed to list images: %s' % (resp.content))
 
 
+@pytest.mark.images_p2
+@pytest.mark.p2
+# This negative test covers images-3-Create Image no data
```

Comment:
Lets use python docstring format for these. https://www.python.org/dev/peps/pep-0257/#toc-entry-6

For example,

```
def test_create_image_no_data(admin_session, harvester_api_endpoints):
"""
Test creating images with no data.

Covers:
* Images-3: Create Image no data
"""
```